app/src/main/res/layout/item_history_card_view.xml

```diff
@@ -0,0 +1,118 @@
+<?xml version="1.0" encoding="utf-8"?>
+<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="@dimen/history_card_view_width"
+    android:layout_height="@dimen/history_card_view_height"
+    android:layout_gravity="center|top"
+    android:background="@drawable/bg_history_cardview"
+    tools:ignore="ContentDescription,HardcodedText">
```

Comment:
Use `ignore` fields only in the specific views where it is necessary. Using it in a parent tag might make you miss some lint warnings inside